### PR TITLE
CLDR-14954 Add Abkhazian locale to CLDR

### PR DESCRIPTION
**Pull request by danielinux7**
CLDR-14954

- [x] This PR completes the ticket.
- [x] Updated PR title and link in previous line to include Issue number.

### CODE REVIEW

**Comment by srl295** [comment]
@danielinux7 should be 1 commit

File: seed/main/ab.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		<languages>
+			<language type="ab">Аԥсшәа</language>
+		</languages>
+	</localeDisplayNames>
+	<characters>
+		<exemplarCharacters>[а б в г {гь} {гә} ӷ {ӷь} {ӷә} д {дә} е ж {жь} {жә} з ӡ {ӡә} и к {кь} {кә} қ {қь} {қә} ҟ {ҟь} {ҟә} л м н о п ԥ р с т {тә} ҭ {ҭә} у ф х {хь} {хә} ҳ {ҳә} ц {цә} ҵ {ҵә} ч ҷ ҽ ҿ ш {шь} {шә} ы ҩ џ {џь} ь ә]</exemplarCharacters>
+		<exemplarCharacters type="auxiliary">[ҕ {ҕь} {ҕә} ҧ {а\u0301} {е\u0301} {и\u0301} {о\u0301} {у\u0301} {ы\u0301}]</exemplarCharacters>
+		<exemplarCharacters type="index">[А Б В Г {ГЬ} {ГӘ} Ӷ {ӶЬ} {ӶӘ} Д {ДӘ} Е Ж {ЖЬ} {ЖӘ} З Ӡ {ӠӘ} И К {КЬ} {КӘ} Қ {ҚЬ} {ҚӘ} Ҟ {ҞЬ} {ҞӘ} Л М Н О П Ԥ Р С Т {ТӘ} Ҭ {ҬӘ} У Ф Х {ХЬ} {ХӘ} Ҳ {ҲӘ} Ц {ЦӘ} Ҵ {ҴӘ} Ч Ҷ Ҽ Ҿ Ш {ШЬ} {ШӘ} Ы Ҩ Џ {ЏЬ}]</exemplarCharacters>
+		<exemplarCharacters type="numbers">[\- ‑ , % ‰ + 0 1 2 3 4 5 6 7 8 9]</exemplarCharacters>
+		<exemplarCharacters type="punctuation">[\- ‐ ‑ – — , ; \: ! ? . … ' ‘ ‚ " “ „ « » ( ) \[ \] \{ \} § @ * / \& #]</exemplarCharacters>
```
Comment (srl295) [inline seed/main/ab.xml:26]:
```suggestion
		<exemplarCharacters type="punctuation">[\- ‐ ‑ – — , ; \: ! ? . … ' ‘ ‚ " “ „ « » ( ) \[ \] \{ \} § @ * / \&amp; #]</exemplarCharacters>
```

**Review by srl295 — COMMENTED** [review]
looks good so far

File: seed/main/ab.xml (hunk 1 above)
Comment (danielinux7) [reply seed/main/ab.xml]: Hello, 
This one is resolved.

**Comment by danielinux7** [comment]
> @danielinux7 should be 1 commit

What should I do now?

**Comment by danielinux7** [comment]
> @danielinux7 should be 1 commit

Would it still be ok or should I close this pull request and create a new one?

**Pull request closed without merging**